cmake/modules/Findthrift.cmake

```diff
@@ -0,0 +1,56 @@
+# This module defines
+#  thrift_LIBRARIES, libraries to link
+#  thrift_INCLUDE_DIR, where to find thrift headers
+#  thrift_COMPILER, thrift compiler executable
+#  thrift_FOUND, If false, do not try to use ant
+
+# prefer the thrift version supplied in thrift_HOME (cmake -Dthrift_HOME then environment)
+find_path(thrift_INCLUDE_DIR
+    NAMES
+        thrift/Thrift.h
+    HINTS
+        ${thrift_HOME}
+        ENV thrift_HOME
+        /usr/local
+        /opt/local
+    PATH_SUFFIXES
+        include
+)
+
+# prefer the thrift version supplied in thrift_HOME
+find_library(thrift_LIBRARIES
+    NAMES
+        thrift libthrift
+    HINTS
+        ${thrift_HOME}
+        ENV thrift_HOME
+        /usr/local
+        /opt/local
+    PATH_SUFFIXES
+        lib lib64
+)
+
+find_program(thrift_COMPILER
```

Comment:
is `thrift_COMPILER` used anywhere ?

Reply:
needed for jaeger's find_package method, IIRC it was dependent on Jaeger some way, can't remember exactly how

Reply:
@ideepika so it is not used anymore. could you please drop it?

Reply:
sure. I tested it locally with example code, jaeger works fine without thrift compiler, but I had to update submodule version: https://github.com/ceph/ceph/pull/38783/commits/512ef7980ee3094a5928914786efb73ce479be23
opened a PR as well: https://github.com/jaegertracing/jaeger-client-cpp/pull/278/files